src/sentry/seer/code_review/webhooks/check_run.py

```diff
@@ -81,31 +78,28 @@ class Config:
         extra = "allow"  # Allow additional fields from GitHub (Pydantic v1 syntax)
 
 
-def preprocess_check_run_event(event: Mapping[str, Any], **kwargs: Any) -> None:
+def preprocess_check_run_event(
+    *, event_type: str, event: Mapping[str, Any], organization: Organization, **kwargs: Any
```

Comment:
The `organization` value is now guaranteed in the `preprocess_webhook_event` function which calls this one.